src/renderer/api-decorator.js

```diff
@@ -426,11 +426,14 @@ limitations under the License.
         let webContentsId = getWebContentsId();
         // Reset state machine values that are set through synchronous handshake between native WebContent lifecycle observers and JS
         options.openfin = true;
-        options.preload = 'preload' in options ? options.preload : winOpts.preload;
+
+        // Force window to be a child of its parent application.
         options.uuid = winOpts.uuid;
+
         // Apply parent window background color to child window when child
         // window background color is unspecified.
         options.backgroundColor = options.backgroundColor || winOpts.backgroundColor;
```

Comment:
Question about this existing code:
Why is this needed here? Is `Window.create` being called? (How, btw?) If so, it inherits options from main window...